bundle/phases/initialize.go

```diff
@@ -39,6 +39,10 @@ func Initialize(ctx context.Context, b *bundle.Bundle) diag.Diagnostics {
 		// If it is an ancestor, this updates all paths to be relative to the sync root path.
 		mutator.SyncInferRoot(),
 
+		// Convert include and exclude in the sync block to use Unix-style slashes.
+		// This is required for the ignore.GitIgnore we use in libs/fileset to work correctly.
+		mutator.SyncIncludeExcludeToSlash(),
+
```

Comment:
Can we fold this into `RewriteSyncPaths` instead of having a dedicated mutator?

Reply:
That's what I initially did but apparently we also modify these paths in .SyncInferRoot and would need to do it there

Reply:
Let's modify both RewriteSyncPath and SyncInferRoot to produce and work with forward slashes?

Reply:
Addressed here https://github.com/databricks/cli/pull/2609/commits/182588e0a50b1b400865d8a8131e351c3ae5a82f